fix a merge the created duplicated content. (#10682)
dotnet/docs: docs/core/tools/csproj.md

### PackageLicenseExpression

> An [SPDX license identifier](https://spdx.org/licenses/) or expression, i.e. `Apache-2.0`.

Reviewer: ```suggestion
An [SPDX license identifier](https://spdx.org/licenses/) or expression. For example, `Apache-2.0`.
```
